content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -44,23 +44,44 @@ This page shows you how to set up a simple Ingress which routes requests to Serv
 
 1. Verify that the NGINX Ingress controller is running
 
-    ```shell
-    kubectl get pods -n kube-system
-    ```
+  {{< tabs name="tab_with_md" >}}
+  {{% tab name="minikube v1.19 or later" %}}
+```shell
+kubectl get pods -n ingress-nginx
+```
+  {{< note >}}This can take up to a minute.{{< /note >}}
+
+Output:
+
+```
+NAME                                        READY   STATUS      RESTARTS    AGE
+ingress-nginx-admission-create-g9g49        0/1     Completed   0          11m
+ingress-nginx-admission-patch-rqp78         0/1     Completed   1          11m
+ingress-nginx-controller-59b45fb494-26npt   1/1     Running     0          11m
+```
+    {{% /tab %}}
+
+  {{% tab name="minikube v1.18.1 or earlier" %}}
```

Comment:
It's not clear what a reader should do if there is ever a minikube v1.18.2 release.

Reply:
@sftim  what should you suggest for keeping version here? should it Be `for version minikube v1.19   or later and minkube versions earlier then minikube 1.19`

Reply:
@sftim Could you please take a look on this! . I have an query regarding how to split the documentation as you suggested for another minikube release

Reply:
@Kartik494 

I would suggest yes, two separate instructions: one for 1.18.1 and earlier and one for 1.19 and newer.

Reply:
@celestehorgan Hi, i have already make two seperations for them as [output](https://github.com/kubernetes/website/pull/28355#issuecomment-859376998) showed by @jihoon-seo above. Is it looks ok to you? or what else suggestion would you to like to suggest here

Thanks!

Reply:
@sftim Could you please take a look!